### ISSUE TITLE
Wrong message reported when toggling native selection in Word

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#16097]
### Steps to reproduce:
* Open a document in MS Word
* Enable browse mode with NVDA+space
* Press NVDA+shift+F10
### Actual behavior:
NVDA says "Application native selection mode not supported in this document"
### Expected behavior:
NVDA should say "Application native selection mode cannot be disabled in this document" or "Application native selection mode cannot be disabled in this application".
Indeed, the native selection is always made in Word when selecting in Browse mode.
Same for Outlook of course.
### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2024.1beta5

#### Windows version:
Windows 10 22H2 (AMD64) build 19045.3930
#### Name and version of other software in use when reproducing the issue:
Office 2016
#### Other information about your system:
N/A
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Not tested but should be the same
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by seanbudd** [comment]
Do you know if this is a recent regression in word or NVDA?

**Comment by seanbudd** [comment]
in just NVDA I mean - seems like a quick fix, but confirming if it should go on the 2024.1 milestone

**Comment by CyrilleB79** [comment]
Native selection feature and `NVDA+shift+f10` shortcuts are new in NVDA 2024.1.

Having this in beta would be clearer to understand this feature while it is launched. But if for whatever reason beta is not suitable, that's not a big problem, because it's not a technical bug from the user's point of view, it's only a matter of communication.

**Comment by CyrilleB79** [comment]
#16129 has solved this issue for word UIA, but not when using object model (still default on Windows 10). Cc @michaelDCurran 

Thus, I reopen this issue until it is also fixed for Word legacy. Sorry.

**Comment by michaelDCurran** [comment]
I can look into this. But do we truly move the real selection in browse 
mode for Word object model? I can't remember. If so, then yes, the 
message needs to be fixed.

**Comment by CyrilleB79** [comment]
> I can look into this. But do we truly move the real selection in browse mode for Word object model? I can't remember. If so, then yes, the message needs to be fixed. Message ID: ***@***.***>

When selecting while in browse mode in a Word / Outlook document, I can confirm you that the selection appears visually, no matter if we use UIA or legacy object model.
